app/presenters/supergroups/guidance_and_regulations.rb

```diff
@@ -0,0 +1,38 @@
+class GuidanceAndRegulations < Supergroup
```

Comment:
Should this be renamed to `GuidanceAndRegulation` for consistency?

Reply:
It should. Cheers!